.github/workflows/test.yml

```diff
@@ -19,21 +19,6 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - ruby: 2.1
-            gemfile: gems/rails3.gemfile
-          - ruby: 2.2
-          - ruby: 2.3
-          - ruby: 2.4
-          - ruby: 2.5
-          - ruby: 2.6
-          - ruby: 2.6
-            gemfile: gems/typhoeus.gemfile
-            test_features: "typhoeus"
-          - ruby: 2.6
-            gemfile: gems/octoshark.gemfile
-          - ruby: 2.6
-            gemfile: gems/rails3.gemfile
```

Comment:
shall we also delete those old gemfiles ?

and bump the minimum requirement in the `scout_apm_ruby.gemspec` ?